lib/amplitude.dart

```diff
@@ -50,7 +40,7 @@ class Amplitude extends _Amplitude {
       event.mergeEventOptions(options);
     }
 
-    return await _channel.invokeMethod("track", jsonEncode(event.toMap()));
```

Comment:
Why is jsonEncode not needed anymore?

Reply:
`jsonEncode()` is what our Flutter SDK used before. But one big problem with JSON string is that, in the Android plugin, `getString()` casts `null` to a string `"null"`. `invokeMethod()` accepts both String and Map data type. So I believe it's easier to get the value at client side and also to debug.